tap_chargebee/schemas/common/customers.json

```diff
@@ -91,6 +91,9 @@
     "cf_company_id": {
       "type": ["null", "integer", "string"]
     },
+    "cf_people_id": {
+      "type": ["null", "integer", "string"]
```

Comment:
why do we need to keep multiple data types? Consider a scenario where the first record is of integer type, and the second record is of string type. Could this potentially result in a column split at the target?

Reply:
I have kept it inline with other id field `cf_company_id` considering that tenant data can be either numeric or alphanumeric.

Reply:
Updated the schema for new fields as suggested.